test/extensions/filters/http/geoip/geoip_filter_integration_test.cc

```diff
@@ -61,6 +61,14 @@ class GeoipFilterIntegrationTest : public testing::TestWithParam<Network::Addres
                                    public HttpIntegrationTest {
 public:
   GeoipFilterIntegrationTest() : HttpIntegrationTest(Http::CodecType::HTTP1, GetParam()) {}
+
+  std::function<absl::string_view(std::string)> header_value =
```

Comment:
Sorry one more nit on this: given that you took it out of the function context, can you just make this a normal class method rather than a lambda? Looksl ike the only context we need is `this`